cdc/entry/main_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package entry
+
+import (
+	"testing"
+
+	"go.uber.org/goleak"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
I see `TestMain` in other PRs, can it be extracted into a function and reused?

Reply:
It is the package level function.  Before all other test-case in any *_test.go,  TestMain will be called firstly to do some initialization work.   

I don't think it can be wrapped.

Reply:
Can we extract following code?

```go
package somepkg

func SetupGoLeak(m *testing.M) {
	opts := []goleak.Option{
		goleak.IgnoreTopFunction("go.etcd.io/etcd/pkg/logutil.(*MergeLogger).outputLoop"),
		goleak.IgnoreTopFunction("go.opencensus.io/stats/view.(*worker).start"),
	}

	goleak.VerifyTestMain(m, opts...)
}

// -------

package otherpkg

import somepkg

func TestMain(m *testing.M) {
    somepkg.SetupGoLeak(m)
}
```

Reply:
But i am not sure  if SetupGoLeak  is so common that need to be wrapped. Some pkgs will add other stack functions.

I will check later.